test/e2e/e2e_test.go

```diff
@@ -23,7 +23,10 @@ import (
 func TestE2EFlow(t *testing.T) {
 	// given
 	// full flow from usersignup with approval down to namespaces creation
-	hostAwait, memberAwait, memberAwait2 := WaitForDeployments(t)
+	awaitilities := WaitForDeployments(t)
+	hostAwait := awaitilities.Host()
+	memberAwait := awaitilities.Member()
```

Comment:
I believe that for the clarity/readability, it would be simpler if we have `Member1()` and `Member2()` methods, aside of the `Member(...)` one

Reply:
I thought about that but I wanted to make it more difficult to get the 2nd member just so it makes it harder to mistakenly use it. 
>The second member cluster in the e2e environment is mainly for testing multi-member specific functionality but is returned by the WaitForDeployments func by default increasing the likelihood that someone uses it accidentally

But if you think it's too confusing we can change it

Reply:
see code a few lines below:
```
signups := CreateMultipleSignups(t, awaitilities, awaitilities.Member(), 5)
```
I believe it would be more explicit if it was:
```
signups := CreateMultipleSignups(t, awaitilities, awaitilities.Member1(), 5)
```
so we now we're creating the 5 users on `Member1`

Reply:
Ah, yes I agree for Member1 that would be good. My comment was more for Member2. I can make the change for Member1 but what do you think about my comments regarding Member2?

Reply:
so maybe having something explicit like `awaitilities.Member(Member1)` and `awaitilities.Member(Member2)` would work? My concern is that ``awaitilities.Member()` without any arg looks a bit strange to read.

Reply:
Ok, fair enough. I went with your first suggestion of using `Member1()` and `Member2()` since it's less verbose than `awaitilities.Member(Member1)`

Done in  https://github.com/codeready-toolchain/toolchain-e2e/pull/388/commits/cbb3fd66aeea0906274f65af34ac03179ff178ad